utils/vpr_io_place.py

```diff
@@ -24,6 +24,21 @@ def __init__(self):
         self.net_to_block = None
         self.net_map = {}
         self.inout_nets = set()
+        self.net_to_pad = set()
+
+    def read_io_loc_pairs(self, blif):
```

Comment:
This needs a docstring explaining what this is doing.  The code looks fine, but a good sentence would be helpful.

Reply:
Maybe add an example of the IO_LOC_PAIRS string?

Reply:
Done